### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Dontmindeme**
Changed reference url from [rawgit](https://rawgit.com/) (which is shutting down) to [jsdelivr](https://www.jsdelivr.com/)

Many thanks for your contribution, we truly appreciate it. We will appreciate it even more, if you make sure that you can say "YES" to each point in this short checklist:

  - You made a small amount of changes (less than 100 lines, less than 10 files) ✔️
  - You made changes related to only one bug (create separate PRs for separate problems) ✔️
  - You are ready to defend your changes (there will be a code review) ✔️
  - You don't touch what you don't understand ✔️
  - You ran the build locally and it passed ✔️

This article will help you understand what we are looking for: http://www.yegor256.com/2015/02/09/serious-code-reviewer.html

Thank you for your contribution!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,19 +19,19 @@ compliant with HTML5 and they will look OK.
 
 Details are here: [yegor256.github.io/tacit](https://yegor256.github.io/tacit/)
 
-Just add it to your HTML (replace `VERSION` with the latest version number,
-which you can find [here](https://github.com/yegor256/tacit/releases)):
+ Just add it to your HTML <!-- (replace `VERSION` with the latest version number,
```
Comment (filfreire) [inline README.md]: @Dontmindeme thanks for the contribution. Kindly follow-up on the CR comments.

Please uncomment this sentence back to it's original state.

File: README.md [hunk 2]
```diff
@@ -19,19 +19,19 @@ compliant with HTML5 and they will look OK.
...
 <!DOCTYPE html>
 <html>
   <head>
-    <link rel="stylesheet" href="https://cdn.rawgit.com/yegor256/tacit/gh-pages/tacit-css-VERSION.min.css"/>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/yegor256/tacit@1.3.6/dist/tacit-css-1.3.6.min.css"/>
   </head>
 </html>
 ```
 
-Or simply download [`tacit-css.min.css`](https://raw.githubusercontent.com/yegor256/tacit/gh-pages/tacit-css.min.css)
```
Comment (filfreire) [inline README.md]: @Dontmindeme please revert this line back to it's original state

The url in this case should be:
```
https://raw.githubusercontent.com/yegor256/tacit/gh-pages/tacit-css.min.css
```

File: README.md [hunk 3]
```diff
@@ -19,19 +19,19 @@ compliant with HTML5 and they will look OK.
...
-which you can find [here](https://github.com/yegor256/tacit/releases)):
+ Just add it to your HTML <!-- (replace `VERSION` with the latest version number,
+which you can find [here](https://github.com/yegor256/tacit/releases)): -->
 
 ```html
 <!DOCTYPE html>
 <html>
   <head>
-    <link rel="stylesheet" href="https://cdn.rawgit.com/yegor256/tacit/gh-pages/tacit-css-VERSION.min.css"/>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/yegor256/tacit@1.3.6/dist/tacit-css-1.3.6.min.css"/>
```
Comment (filfreire) [inline README.md]: @Dontmindeme please use this URL instead:
```
https://cdn.jsdelivr.net/gh/yegor256/tacit@VERSION/dist/tacit-css-VERSION.min.css
```

File: README.md (hunk 2 above)
Comment (filfreire) [reply README.md]: @Dontmindeme forget this one comment, please change the url to:
```https://cdn.jsdelivr.net/gh/yegor256/tacit@gh-pages/tacit-css.min.css```

**Comment by Dontmindeme** [comment]
Done. ?

**Comment by Dontmindeme** [comment]
Done. ?

**Comment by Dontmindeme** [comment]
Happy New Year 2019

**Comment by filfreire** [comment]
@Dontmindeme sorry, forgot about following-up to this PR. Will do so ASAP.

**Comment by Dontmindeme** [comment]
No Problem.

File: README.md (hunk 2 above)
Comment (Dontmindeme) [reply README.md]: ...

**Comment by filfreire** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/tacit#174` is not assigned, can't get performer

**Comment by 0crat** [comment]
This job is not in scope

**Comment by filfreire** [comment]
@Dontmindeme PR is merged. Thank you 👍

**Pull request merged**